CMakeLists.txt

```diff
@@ -5,46 +5,69 @@ project(match-theory VERSION 0.1.0)
 set(CMAKE_CXX_STANDARD 17)
```

Comment:
Somehow I've never noticed this but this file uses 5 space indentation for some reason.

Reply:
Probably a copy and paste error. I'll address it.